utils/create_dummy_models.py

```diff
@@ -71,6 +72,67 @@
 TARGET_VOCAB_SIZE = 1024
 
 
+# This list contains the model architectures for which a tiny version could not be created.
+# Avoid to add new architectures here - unless we have verified carefully that it's (almost) impossible to create them.
+# One such case is: no model tester class is implemented for a model type (like `MT5`) because its architecture is
+# identical to another one (`MT5` is based on `T5`), but trained on different datasets or with different techniques.
+UNCONVERTIBLE_MODEL_ARCHITECTURES = {
```

Comment:
Main change 1:  record models that could not be converted to tiny versions (so don't run them in the future)